src/clj/chronograph/db/acl.clj

```diff
@@ -1,24 +1,8 @@
 (ns chronograph.db.acl
-  (:require [chronograph.db.core :as db]
-            [next.jdbc.sql :as sql]
-            [chronograph.utils.time :as time])
-  (:import (org.postgresql.util PGobject)))
+  (:require [chronograph.db.core :as db]))
 
-(defn create! [tx {:keys [user-id organization-id role]}]
-  (let [now (time/now)]
-    (sql/insert! tx
-                 :acls
-                 {:user-id user-id
-                  :organization-id organization-id
-                  :role (doto (PGobject.)
-                          (.setType "user_role")
-                          (.setValue role))
-                  :created-at now
-                  :updated-at now}
-                 db/sql-opts)))
+(def create! (partial db/create! :acls))
```

Comment:
Not related to this PR, but I've been increasingly uncomfortable about this pattern of aliasing db functions with a `def`, because it takes away the documentation of argument lists.

I find myself having to jump 2/3 namespaces to understand what the function actually needs. 

Writing a wrapper function would be much better, IMO. 

WDYT?

Reply:
Specific to using `partial`. It muddies the water further by completely obscuring how many arguments the partial application can consume, because Clojure functions are variadic.

No tasty currying for us, sadly.